main.py

```diff
@@ -9,15 +9,25 @@
 import sys
 import time
 import traceback
+from multiprocessing import Queue, Process
```

Comment:
I would do `import multiprocessing` and use `multiprocessing.Queue` etc. further down; I think that makes it clearer what kind of queue is being used